Monika After Story/game/script-topics.rpy

```diff
@@ -18041,3 +18041,76 @@ label monika_overton:
     m 1eua "I {i}could{/i} finish this story right up to Current Norm, but I just wanted to describe it at a basic level in order to convey an example of how it can work."
     m 1huu "Thanks for listening~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_upset_apologize",
+            category=['you','monika'],
+            prompt="Something you did upset me.",
+            pool=True,
+            unlocked=True
+        )
+    )
+
+label monika_upset_apologize:
+    $ mas_loseAffection()
+    if mas_isMoniBroken():
+        m "...{w=1}Oh."
```

Comment:
There should be a space after ellipsis; besides, for consistency, it's better to use `w=0.3` for pauses like that.